src/main/java/at/hannibal2/skyhanni/config/features/foraging/ForagingConfig.kt

```diff
@@ -64,4 +64,18 @@ class ForagingConfig {
     @FeatureToggle
     @OnlyModern
     var mutePhantoms = true
+
+    @Expose
+    @ConfigOption(name = "Mute Tree Breaking", desc = "Mutes the sound of the tree fully breaking.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    @OnlyModern
+    var muteTreeBreaking = true
+
+    @Expose
+    @ConfigOption(name = "Also on Galatea", desc = "Also mutes tree breaking sounds on Galatea.")
+    @ConfigEditorBoolean
+    @FeatureToggle
```

Comment:
this feature toggle here can go. the main toggle is "mute tree breaking", and "also on galatea" is only a sub toggle